fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java

```diff
@@ -80,4 +81,34 @@ public void sendToUserAccount(final EmailDetail emailDetail, final String unenco
             throw new PlatformEmailSendException(e);
         }
     }
+
+    @Override
+    public void sendDefinedEmail(EmailData emailDetails) {
```

Comment:
Modify the existing method sendToUserAccount to make use of this method instead of having duplicate implementation.